### DISCUSSION TITLE
I deleted a trigger in developer platform, but impact to maker

### DISCUSSION THREAD

**Discussion by marioekaputta** [bahasa-ai/otomatis-guidance#29] [Q&A]
So I deleted a trigger in my developer platform, but I forgot to turn off the scenario in my maker platform. Now the scenario deleted the trigger itself.
![image](https://user-images.githubusercontent.com/71392755/134622915-536808f4-71f5-4640-9ae3-d9b56c79d865.png)
In the image above, it was between the google form and gmail. Now it's only gmail.

The problem is, I can't turn off this scenario
![image](https://user-images.githubusercontent.com/71392755/134622993-1cb771ca-e3c2-4d78-8746-cb4233b4b5ef.png)
In the image above, that's what happens when I try to turn off the scenario.

And I can't modify the scenario as well
![image](https://user-images.githubusercontent.com/71392755/134623081-55ffb03a-dc70-4856-8a32-58adb16b775a.png)
In the image above, that's what happens when I try to modify the scenario.

And of course, I can't delete the scenario until it's turn off. How can I resolve this?

**Comment by mgilangjanuar** [comment]
Thank you for reporting, please wait until we fix it, we'll get back to you after it resolved.
